libraries/ms-common/inc/can_ack.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+// CAN ACK handling
+//
+// ACK request objects are provided with critical messages.
+// They define a callback that should be run when the ACK status is updated.
+// The ACK status is updated when
+// a) We receive an ACK over CAN
+// b) The timer expires and we timeout the ACK
+//
+// If the ACK has timed out or we've received the expected number of ACKs, we remove the ACK request
+// from the list. We use an object pool and an array of pointers to do so efficiently.
+#include "can_msg.h"
+#include "objpool.h"
+#include "status.h"
+#include "soft_timer.h"
+
+// ACK timeout - transit time + computation time
```

Comment:
I mean it was meant to be a dash and more of a note than an equation. It's more like "should account for".